shared/src/main/kotlin/org/javacs/kt/classpath/MavenClassPathResolver.kt

```diff
@@ -122,7 +190,8 @@ data class Artifact(
     val packaging: String?,
     val classifier: String?,
     val version: String,
-    val scope: String?
+    val scope: String?,
+    var source: Boolean
```

Comment:
Could you update the `MavenArtifactParsingTest` and (ideally) add a test case that verifies that source JAR artifact parse correctly?